src/Rings/mpoly-ideals.jl

```diff
@@ -2003,3 +2003,68 @@ function Base.show(io::IO, I::MPolyIdeal)
     print(io, "Ideal ", _get_generators_string_one_line(I))
   end
 end
+
+
+
+
+_pluecker_sgn(a::Vector{Int},b::Vector{Int},t::Int) = count(z->z>t,a)+count(z->z<t,b)
+    
+@doc raw"""
+    flag_pluecker_ideal(dimensions::Vector{Int},n::Int)
+
+Returns generators of the defining ideal for the flag variety $\text{Fl}((d_1,\dots,d_k),n)$, where $(d_1,\dots,d_k) =$`dimensions` denotes the rank, with $d_j\leq n-1$. That is, the space of $k$-step flags of linear subspaces $V_1\subset\dots\subset V_k$ in $\mathbb{C}^n$, where $\text{dim}(V_j) = d_{j}$.You can obtain the generators for the \emph{complete flag variety}$ by taking `dimensions` $=(1,\dots,n-1)$ and `n`$=n$. 
+
+# Example
+Complete flag variety $\text{Fl}((1,2,3),4)$. 
+```jldoctest
+julia> flag_pluecker_ideal([1,2,3],4)
+Ideal generated by
+  x[[1]]*x[[3, 4]] - x[[3]]*x[[1, 4]] + x[[4]]*x[[1, 3]]
+  -x[[1, 4]]*x[[2, 3]] + x[[2, 4]]*x[[1, 3]] - x[[3, 4]]*x[[1, 2]]
+  x[[2]]*x[[3, 4]] - x[[3]]*x[[2, 4]] + x[[4]]*x[[2, 3]]
+  x[[1]]*x[[2, 4]] - x[[2]]*x[[1, 4]] + x[[4]]*x[[1, 2]]
+  x[[1]]*x[[2, 3]] - x[[2]]*x[[1, 3]] + x[[3]]*x[[1, 2]]
+  x[[1]]*x[[2, 3, 4]] - x[[2]]*x[[1, 3, 4]] + x[[3]]*x[[1, 2, 4]] - x[[4]]*x[[1, 2, 3]]
+  -x[[1, 4]]*x[[1, 2, 3]] + x[[1, 3]]*x[[1, 2, 4]] - x[[1, 2]]*x[[1, 3, 4]]
+  -x[[2, 4]]*x[[1, 2, 3]] + x[[2, 3]]*x[[1, 2, 4]] - x[[1, 2]]*x[[2, 3, 4]]
+  -x[[3, 4]]*x[[1, 2, 3]] - x[[1, 3]]*x[[2, 3, 4]] + x[[2, 3]]*x[[1, 3, 4]]
+  -x[[1, 4]]*x[[2, 3, 4]] + x[[2, 4]]*x[[1, 3, 4]] - x[[3, 4]]*x[[1, 2, 4]]
+
+```
+# Example 2
+
+Flag variety $\text{Fl}((1,3),4)$.
+```jldoctest
+julia> flag_pluecker_ideal([1,3],4)
+Ideal generated by
+  x[[1]]*x[[2, 3, 4]] - x[[2]]*x[[1, 3, 4]] + x[[3]]*x[[1, 2, 4]] - x[[4]]*x[[1, 2, 3]]
+```
+"""
+function flag_pluecker_ideal(dimensions::Vector{Int},n::Int)
+    
+    N = collect(1:n)
+    
+    L = reduce(union,[subsets(N,d) for d in dimensions])
```

Comment:
```suggestion
    L = reduce(union,[subsets(n,d) for d in dimensions])
```
is better (also below, and in the end you can get rid of `N`)